paasta_tools/cli/cmds/start_stop_restart.py

```diff
@@ -169,6 +182,13 @@ def paasta_start_or_stop(args, desired_state):
 
     invalid_deploy_groups = []
     marathon_message_printed, chronos_message_printed = False, False
+
+    if args.clusters is None or args.instances is None:
```

Comment:
Thank you. Not only will this help make the tests pass, but it will be less annoying to users and scripts